dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -30,9 +32,22 @@ - (void)openURLInSafari:(NSURL *)URL;
 @implementation DMPlayerViewController
 
 - (void)dealloc {
-  UIWebView *webView = (UIWebView *)self.view;
-  webView.delegate = nil;
```

Comment:
@JanGorman To be sure it won't happen again my implementation changes the way UIWebView is used. I've tested sample and my application - nothing changed, but bug should be fixed.